### PR TITLE
add link to "getting started with NI-DAQmx" to additional documentation

### LINKED ISSUE #166: add overview to documentation
The documentation basically just documents each function... but it's hard to understand without some sort of high level overview. Like how the timing system works, what a task is, the limitations on having separate tasks, etc. I assume this stuff in the c documentation? So far I haven't found it.

### PR DESCRIPTION
**Pull request by ggggggggg**
fixes #166

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -172,6 +172,8 @@ Documentation is available `here <http://nidaqmx-python.readthedocs.io>`_.
 Additional Documentation
 ========================
 
+Refer to the `Getting Started with NI-DAQmx <https://www.ni.com/en-us/support/documentation/supplemental/06/getting-started-with-ni-daqmx--main-page.html>`_ page for more detailed documentation of the C and labview interfaces and higher level concepts such as Tasks. The python API follows the C API names relatilvey closely.
```
Comment (maxxboehme) [inline README.rst:175]: Capitalize LabVIEW and Python.

Also maybe reword the first sentence to something like. 
... for more detailed documentation of the C and LabVIEW interfaces along with higher level API concepts such as Tasks.

Comment (ggggggggg) [reply README.rst]: done

**Review by maxxboehme — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -172,6 +172,8 @@ Documentation is available `here <http://nidaqmx-python.readthedocs.io>`_.
 Additional Documentation
 ========================
 
+Refer to the `Getting Started with NI-DAQmx <https://www.ni.com/en-us/support/documentation/supplemental/06/getting-started-with-ni-daqmx--main-page.html>`_ page for more detailed documentation of the C and LabVIEW interfaces along with higher level API concepts such as Tasks. The Python API follows the C API names relatilvey closely.
```
Comment (zhindes) [inline README.rst:175]: The convention appears to wrap at 80 characters if possible. That link isn't gonna work great, but at least word wrap after the link and then for the following blurb, I guess.

**Review by zhindes — CHANGES_REQUESTED** [review]

**Comment by zhindes** [comment]
Closing due to inactivity
